src/rhsmlib/dbus/facts/com.redhat.RHSM1.Facts.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
```

Comment:
Gathering the facts requires root privileges.  The end goal is to have subscription-manager not need root access for normal stuff (e.g. you should need root access just to see your attached subscriptions).  If you're doing something that requires facts, we request the info from the facts service that DBus Daemon starts as root.